### PR TITLE
Flaw References and Comments support

### PR DESCRIPTION
**Pull request by JakubFrejlach**
This PR introduces the new subresources (nested resources) to be used in session operations.

It also brings support for Flaw References and Flaw Comments.

### CODE REVIEW

File: osidb_bindings/constants.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 from typing import Dict, List
 
 OSIDB_API_VERSION: str = "v1"
-OSIDB_BINDINGS_USERAGENT: str = "osidb-bindings-3.1.0"
+OSIDB_BINDINGS_VERSION: str = "3.1.0"
+OSIDB_BINDINGS_USERAGENT: str = f"osidb-bindings-{OSIDB_BINDINGS_VERSION}"
```
Comment (JimFuller-RedHat) [inline osidb_bindings/constants.py:5]: good idea for corgi as well

File: osidb_bindings/session.py [hunk 2]
```diff
@@ -297,26 +330,30 @@ def update(self, id, form_data: Dict[str, Any]):
...
+                **kwargs,
             )
         else:
             raise OperationUnsupported(
                 'Operation "update" is not supported for the '
                 f'"{self.resource_name}" resource.'
             )
 
-    def delete(self, id):
+    def delete(self, id, *args, **kwargs):
```
Comment (JimFuller-RedHat) [inline osidb_bindings/session.py:346]: a (very) random and unrelated observation - are cascade deletes in data service layer also removing comments/references ?

**Comment by JimFuller-RedHat** [comment]
LGTM

**Review by JimFuller-RedHat — APPROVED** [review]
LGTM - and if you might bump related python modules aka github vuln reports ;)

File: osidb_bindings/session.py (hunk 2 above)
Comment (JakubFrejlach) [reply osidb_bindings/session.py:346]: Yes, cascade deletes are set.

File: osidb_bindings/constants.py (hunk 1 above)
Comment (JakubFrejlach) [reply osidb_bindings/constants.py:5]: Basically I did that so I can implement later implement the version check mechanism for this [issue](https://github.com/RedHatProductSecurity/osidb-bindings/issues/11).

**Pull request merged**